### ISSUE TITLE
update unicode "₊" separator to "."

### ISSUE THREAD

**Issue by isaacsas** [SciML/Catalyst.jl#949]
https://github.com/SciML/ModelingToolkit.jl/pull/2798

We need to propagate this throughout Catalyst where needed (probably some tutorials, tests, and the `symmap_to_varmap` code). Also need to update to depend on MTK 9.18 or later only.

**Comment by ChrisRackauckas** [comment]
https://github.com/SciML/Catalyst.jl/pull/951

**Comment by isaacsas** [comment]
There are still some tests and docs that explicitly use "₊". We need to update them whenever MTK decides to change the separator.

**Comment by ChrisRackauckas** [comment]
Just use the general separator const?

**Comment by isaacsas** [comment]
Yup that is all that needs to be done for the tests, but the doc example will require a bit more tweaking.

**Comment by isaacsas** [comment]
stale
